environments/gloo-gateway/core/istio/lifecyclemanager/base/active/ilcm-mgmt.yaml

```diff
@@ -65,6 +65,9 @@ spec:
           global:
             meshID: mesh1
             network: "network1"
+        global:
```

Comment:
We need this on all Istio Operator specs in order for the OTel collector to correctly identify which metrics come from which cluster.